src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/FeatureToggleServiceImpl.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.cwrdapi.service.impl;
+
+import com.launchdarkly.sdk.LDUser;
+import com.launchdarkly.sdk.server.LDClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.cwrdapi.service.FeatureToggleService;
+
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.PostConstruct;
+
+@Service
+public class FeatureToggleServiceImpl implements FeatureToggleService {
+
+    @Autowired
+    private final LDClient ldClient;
+
+    @Value("${launchdarkly.sdk.environment}")
+    private String environment;
+
+    private final String userName;
+
+    private Map<String, String> launchDarklyMap;
+
+    @Autowired
+    public FeatureToggleServiceImpl(LDClient ldClient, @Value("${launchdarkly.sdk.user}") String userName) {
+        this.ldClient = ldClient;
+        this.userName = userName;
+    }
+
+    @PostConstruct
+    public void mapServiceToFlag() {
+        launchDarklyMap = new HashMap<>();
+        launchDarklyMap.put("CaseWorkerRefController.fetchCaseworkersById",
+                "crd-fetch-caseworker-profile-by-id");
```

Comment:
the flag name needs to change... it should be one feature toggle flag for both the POST and GET APIs in CRD

Reply:
Acknowledged. Will update the code to use single LD flag for all endpoints in CRD.